content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -82,16 +82,25 @@ The project containing the sample CorDapp opens.
 1. Navigate to the root directory of the project from the command line.
 
 2. Configure the network:
-
-`corda-cli network config docker-compose solar-system`
+```
+corda-cli network config docker-compose solar-system
+```
 
 3. Run a Gradle command ([or run a Gradle task in IntelliJ](https://www.jetbrains.com/help/idea/work-with-gradle-tasks.html#gradle_tasks)) to build the CorDapp and generate the `.cpb` file:
 
-  `./gradlew build`
+```
+./gradlew build
+```
 
 This command builds the CorDapp package (`.cpk`) files. Most CorDapps have two files: a `contracts` file and a `workflows` file. The command also builds the CorDapp package bundle (`.cpb`).
 
-4. Deploy the network using `corda-cli` and `docker-compose`:
+4. Compile your `.cpk` files into a single `.cpb` file using the CorDapp Builder CLI:
```

Comment:
@peterli-r3 This is the step that I mentioned to you. We didn't have this step before, so when I ran through the tutorial there was no cpb. When I used this command, then deployed the network everything worked.

Reply:
Let me know what you think.